web-common/src/features/dashboards/time-series/multiple-dimension-queries.ts

```diff
@@ -0,0 +1,80 @@
+// Throw away code for fetching timeseries data for individual dimension values
```

Comment:
"Throw away code"?

Reply:
We would be replacing this query with a single aggregation query on the backend in a couple of releases.